infra/data-pipeline.yml

```diff
@@ -1290,8 +1290,6 @@ tasks:
         - train-test-group-select
         - train-needsdiagnosis
       payload:
-        capabilities:
-          privileged: true
```

Comment:
You're going to have to keep these lines so that d2g adds in the `--privileged` flag to podman run.